cpp/ql/test/library-tests/ir/ir/raw_ir.expected

```diff
@@ -7078,11 +7188,29 @@ ir.cpp:
 #-----|   Goto -> Block 6
 
 # 1244|   Block 6
-# 1244|     v1244_1(void)  = NoOp                       : 
-# 1240|     v1240_8(void)  = ReturnIndirection[dynamic] : &:r1240_6, ~m?
-# 1240|     v1240_9(void)  = ReturnVoid                 : 
-# 1240|     v1240_10(void) = AliasedUse                 : ~m?
-# 1240|     v1240_11(void) = ExitFunction               : 
+# 1244|     v1244_1(void)            = NoOp                            : 
+# 1244|     r1244_2(glval<String>)   = VariableAddress[c]              : 
+# 1244|     r1244_3(glval<unknown>)  = FunctionAddress[~String]        : 
+# 1244|     v1244_4(void)            = Call[~String]                   : func:r1244_3, this:r1244_2
+# 1244|     mu1244_5(unknown)        = ^CallSideEffect                 : ~m?
+# 1244|     v1244_6(void)            = ^IndirectReadSideEffect[-1]     : &:r1244_2, ~m?
+# 1244|     mu1244_7(String)         = ^IndirectMayWriteSideEffect[-1] : &:r1244_2
+# 1244|     r1244_8(glval<String>)   = VariableAddress[b]              : 
+# 1244|     r1244_9(glval<unknown>)  = FunctionAddress[~String]        : 
+# 1244|     v1244_10(void)           = Call[~String]                   : func:r1244_9, this:r1244_8
+# 1244|     mu1244_11(unknown)       = ^CallSideEffect                 : ~m?
+# 1244|     v1244_12(void)           = ^IndirectReadSideEffect[-1]     : &:r1244_8, ~m?
+# 1244|     mu1244_13(String)        = ^IndirectMayWriteSideEffect[-1] : &:r1244_8
+# 1244|     r1244_14(glval<String>)  = VariableAddress[a]              : 
+# 1244|     r1244_15(glval<unknown>) = FunctionAddress[~String]        : 
+# 1244|     v1244_16(void)           = Call[~String]                   : func:r1244_15, this:r1244_14
+# 1244|     mu1244_17(unknown)       = ^CallSideEffect                 : ~m?
+# 1244|     v1244_18(void)           = ^IndirectReadSideEffect[-1]     : &:r1244_14, ~m?
+# 1244|     mu1244_19(String)        = ^IndirectMayWriteSideEffect[-1] : &:r1244_14
```

Comment:
I don't think this is correct. `a`, `b` and `c` are all static locals. So their destructors should not be called at the end of the scope. Ideally, we should have some kind of new `IRFunction` that calls these destructors at the end of `main`, but let's leave that aside for now.

As a start, I think we should simply not emit these destructors for now.

Reply:
Oh, good point. I missed that those were static when I was looking over the results.